docs/getting-started.md

```diff
@@ -122,6 +122,17 @@ command line.
 
 ## Configuring your system
 
+### Add user details to your git config
+To create and publish projects to the Radicle network, you must associate your commits with an identity by setting a username in Git.
+
+You can do so by running the following command:
+
+```
+git config --global user.name "YOUR NAME"
+```
+For more help, refer to the relevant [Git documentation][gd].
+
+### Add the remote helper to your shell
 To publish code to the Radicle network via the Upstream client, you first have
 to add the [Radicle git remote helper][rg] to your shell configuration.
```

Comment:
There is no section about the remote helper in the FAQ.

Reply:
Yup. Will fix

Reply:
I decided to remove the link in my fix, since there isn't a good place to link it in the FAQ